changelog/3307.feature.rst

```diff
@@ -0,0 +1,3 @@
+pytest should not change the log level of the root logger
+when the ``log-level`` parameter is set with value
+``LOG_LEVEL>root_logger_level``.
```

Comment:
-> when the `log-level` parameter has greater numeric value than that of the level of the root logger.

Reply:
Done.